src/index.html

```diff
@@ -5,6 +5,168 @@
+        <h2></h2> -->
+        <!-- glide -->
+      <!-- </div>
+      <div class="favorites-view__saved-week-two-recipes">
+        <h2></h2> -->
+        <!-- glide -->
+      <!-- </div>
+      <div class="favorites-view__saved-week-three-recipes">
+        <h2></h2> -->
+        <!-- glide -->
+      <!-- </div>
+      <div class="favorites-view__saved-week-four-recipes">
+        <h2></h2> -->
+        <!-- glide -->
+      <!-- </div> -->
+
+    </section>
+    <section class="weekly-recipes-view">
+      <nav class="nav-bar">
+        <div class="nav-tabs">
+          <button type="button" class="nav-tabs__all-recipes"></button> all recipes
+          <button type="button" class="nav-tabs__favorites"></button> favourites
+          <button type="button" class="nav-tabs__this-week"></button> this week
+        </div>
+        <div class="search-bar">
+          <label for="searchInput"></label> search bar
+          <input type="search" name="searchInput" class="search-bar__search-input" value=""> search bar
+          <button type="button" class="search-bar__initiate-search"></button> search button
+          <button type="button" class="search-bar__home-button"></button> home
+        </div>
+      </nav>
+      <h1></h1> <!-- weekly recipes -->
+      <div class="weekly-recipes-view__week-one">
+        <h2>week of ...</h2>
+        <!-- glide -->
+      </div>
+      <div class="weekly-recipes-view__week-two">
+        <h2>week of ...</h2>
+        <!-- glide -->
+      </div>
+      <div class="weekly-recipes-view__week-three">
+        <h2>week of ...</h2>
+        <!-- glide -->
+      </div>
+      <div class="weekly-recipes-view__week-four">
+        <h2>week of ...</h2>
+        <!-- glide -->
+      </div>
+    </section>
+    <section class="single-recipe-view">
+      <nav class="nav-bar">
+        <div class="nav-tabs">
+          <button type="button" class="nav-tabs__all-recipes"></button> all recipes
+          <button type="button" class="nav-tabs__favorites"></button> favourites
+          <button type="button" class="nav-tabs__this-week"></button> this week
+        </div>
+        <div class="search-bar">
+          <label for="searchInput"></label> search bar
+          <input type="search" name="searchInput" class="search-bar__search-input" value=""> search bar
+          <button type="button" class="search-bar__initiate-search"></button> search button
+          <button type="button" class="search-bar__home-button"></button> home
+        </div>
+      </nav>
+      <!-- recipes view -->
+      <h1>recipe name</h1>
+      <button type="button" class="single-recipe-view__add-to-favorites-button"></button> add to favourites
+      <button type="button" class="single-recipe-view__add-to-week-button"></button> cook this week
+      <div class="single-recipe-view__ingredients">
+        <h2>ingredients title</h2>
+        <text class="ingredients__ingredient-list">ingredients</text>
```

Comment:
It would be cool to have a text tag, but let's remember to change these to p tags.